apps/shared/form/Textfield.js

```diff
@@ -16,6 +16,7 @@ const Textfield = props => {
     description,
     pattern,
     maxLength,
+    showMaxLength,
```

Comment:
sorry - what is this trying to achieve?

Reply:
The designs for this response form included "100 characters maximum" text below the input fields - the bool is to toggle this display (so adding it doesn't influence existing Textfield uses).

It's probably safe to just include in other uses but my approach for this was to avoid influencing how shared components works elsewhere.